bindings/nodejs/lib/types/wallet/bridge/wallet.ts

```diff
@@ -9,7 +9,7 @@ import type {
 } from '../../client';
 import type {
     SendParams,
-    SendNativeTokensParams,
+    SendNativeTokenParams,
```

Comment:
SendNativeTokensParams still has `nativeTokens: [TokenId, u256][];`, `[]` should be removed

Reply:
Same in Python

Reply:
Nvm, I reviewed the last commit :trollface:

Reply:
Hmm it doesn't?

Reply:
Ah, no worries